### PR TITLE
Add new speaker: Katrina Owen

### PR DESCRIPTION
**Pull request by olivierobert**
## What happened

- Add new speaker entry for Katrina Owen 
- Remove speaker entry for Steven Baker
- Update the schedule with Katrina Owen's talk
 
## Insight

Steven Baker cancelled his talk, so Katrina Owen has jumped in as a replacement speaker. 

I re-ordered the list of speakers alphabetically and standardized the size of some speakers' headshots.
 
## Proof Of Work

|Speaker List|Schedule|
|-|-|
|image|image|

### CODE REVIEW

File: src/_data/speakers.yml [hunk 1]
```diff
@@ -97,40 +116,45 @@
...
-  last_name: Jain
+  website: https://dugiahuy.com
+
+- id: katrina-owen
+  first_name: Katrina
+  last_name: Owen
   type: speaker
-  social_url: https://twitter.com/jainrishi15
+  photo_url: /images/speakers/katrina-owen.jpg
+  bio: Katrina has worked as a professional software developer for 18 years. She is the co-author of <a href="https://sandimetz.com/99bottles">99 Bottles of OOP</a>. Katrina is probably most well known for creating <a href="https://exercism.org/">Exercism</a> - a free, not-for-profit coding education platform powered by a wonderful volunteer community, that serves over 1.5 million users.
```
Comment (matthewmayer) [inline src/_data/speakers.yml:135]: We don't normally include hyperlinks on bios

**Review by matthewmayer — CHANGES_REQUESTED** [review]

File: src/_data/speakers.yml (hunk 1 above)
Comment (olivierobert) [reply src/_data/speakers.yml:135]: It works, though. I made a fix in one if the component: https://github.com/bangkokrb/rubyconfth/pull/335/files#diff-3fe6074553b6b502b27879f0ba145d5cb807e5d6e7c483adcd9ec89188621cbbL23

Comment (matthewmayer) [reply src/_data/speakers.yml:135]: just dont want the other speakers to complain they dont have hyperlinks :D

Comment (olivierobert) [reply src/_data/speakers.yml]: Fair point. Let's got with equality. I have removed the links in 86a40ef ✅

**Review by citizen428 — APPROVED** [review]

**Pull request merged**